compiler/test/formatter_outputs/records.gr

```diff
@@ -110,3 +110,19 @@ let s = { // comment 1
   var: A, // end line comment
   str: "",
 }
+
+let y = { ...x, bar: 2 }
+
+let y = { ...x, bar: 2 }
+
+let y = {
+  ...x,
+  longlonglongnamenamename2: 12345,
+  longlonglongnamenamename3: 12345,
+}
+
+let y = { /* comment 1 */ /* comment 2 */ /* comment 3 */ /* comment 4 */
```

Comment:
This seems wrong

Reply:
This is just how we format this currently; fixing it is probably out of scope of this PR. Can make an issue to address though.

Reply:
I thought @marcusroberts literally just fixed these?

Reply:
If you remove the `...` this is the behavior on main.

Reply:
I'll take a look

Reply:
I've raised an issue for this https://github.com/grain-lang/grain/issues/1575 

It is on main though, so not a problem with this PR

Reply:
Fixed in #1578

Reply:
Thanks for the quick fix! I can rebase this branch off of main when your PR gets merged.

Reply:
Sweet, thanks!